hack/cloud-image-uploader/azure-disk-template.json

```diff
@@ -45,7 +45,7 @@
             "storageAccountId": "[concat('/subscriptions/7f739b7d-f399-4b97-9a9f-f1962309ee6e/resourceGroups/SideroGallery/providers/Microsoft.Storage/storageAccounts/', parameters('storage_account'))]",
             "sourceUri": "[concat('https://siderogallery.blob.core.windows.net/images/talos/', parameters('vhd_name'))]"
           },
-          "diskSizeGB": 10,
+          "diskSizeGB": 8,
```

Comment:
Disk size is chaned to 8GB to be within the limits of the P2 Azure disk tier. The price of storing per disk doubles when moving to P3 (up to 16GB). The disk size of Talos cloud images was also changed to match in: [https://github.com/siderolabs/talos/pull/7252](https://github.com/siderolabs/talos/pull/7252).